views/dropdowns/pingpoll_time.py

```diff
@@ -12,7 +12,7 @@ def __init__(self):
         ]
 
         super().__init__(
-            placeholder="Please select a time",
+            placeholder="Please select how long the PingPoll should stay.",
```

Comment:
Please remove the dot at the end of the sentence
```suggestion
            placeholder="Please select how long the PingPoll should stay",
```